piptools/resolver.py

```diff
@@ -161,7 +161,9 @@ def resolve(self, max_rounds: int) -> set[InstallRequirement]:
     def resolve_hashes(
         self, ireqs: set[InstallRequirement]
     ) -> dict[InstallRequirement, set[str]]:
-        """Find acceptable hashes for all of the given ``InstallRequirement``s."""
+        """
+        Find acceptable hashes for all of the given ``InstallRequirement``'s.
```

Comment:
This change is incorrect. It's not `'s` but just a plural `s`.

Reply:
I wouldn't have added that if Sphinx hadn't given me this strange error:

```
/home/user/repos/github/pip-tools/.tox/build-docs/lib/python3.9/site-packages/piptools/resolver.py:docstring of piptools.resolver.BaseResolver.resolve_hashes:1: WARNING: Inline literal start-string without end-string.
```

Reply:
Either way, the grammar is broken. RST parser can be tricked into merging parts by adding an escaped whitespace:
```suggestion
        Find acceptable hashes for all of the given ``InstallRequirement``\ s.
```

If that doesn't work, the only thing left is rephrasing. There are other hacks, I think, but they are rather complicated and not worth it.